src/paperoni/__main__.py

```diff
@@ -782,7 +836,12 @@ def run(self):
             }
         with gifnoc.overlay(overrides):
             app = create_app()
-            uvicorn.run(app, host=self.host, port=self.port, reload=self.reload)
+            uvicorn.run(
+                app,
+                host=config.server.host if self.host is None else self.host,
+                port=config.server.port if self.port is None else self.port,
```

Comment:
```suggestion
                host=self.host or config.server.host
                port=self.port or config.server.port,
```

Reply:
Specifying port 0 is legal, it will pick a random free port.